src/Core/src/Platform/iOS/HandlerExtensions.cs

```diff
@@ -6,11 +6,17 @@ namespace Microsoft.Maui
 {
 	public static class HandlerExtensions
 	{
+		public static UIViewController ToUIViewController(this IView view, IMauiContext context) => new ContainerViewController { CurrentView = view, Context = context };
```

Comment:
Wonder if this extension method name could be something more similar to the android one or vice-versa